packages/expo-font/ios/EXFont/EXFontLoaderProcessor.m

```diff
@@ -6,10 +6,33 @@
 #import <EXFont/EXFontManager.h>
 #import <objc/runtime.h>
 
-static NSString *exponentPrefix = @"ExpoFont-";
+@interface EXFontLoaderProcessor ()
+
+@property (nonatomic, copy) NSString *fontFamilyPrefix;
+@property (nonatomic, strong) EXFontManager *manager;
+
+@end
 
 @implementation EXFontLoaderProcessor
 
+- (instancetype)initWithFontFamilyPrefix:(NSString *)prefix
+                              andManager:(EXFontManager *)manager
+{
+  if (self = [super init]) {
+    _fontFamilyPrefix = prefix;
+    _manager = manager;
+  }
+  return self;
+}
+
+- (instancetype)initWithManager:(EXFontManager *)manager
+{
+  if (self = [super init]) {
```

Comment:
Call `initWithFontFamilyPrefix:nil manager:manager` instead -- Cocoa (esp with Swift) has the notion of "designated initializers" that the compilers ensures are called. In this case, `-[initWithFontFamilyPrefix: manager:]` would be the designated initializer.